### PR TITLE
feat(debug): add teamsfx-run-utils package

### PR DESCRIPTION
**Pull request by kuojianlu**
E2E TEST: N/A

### CODE REVIEW

File: packages/run-utils/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    throw new Error(`${projectPath} does not exist.`);
+  }
+
+  if (!(await fs.pathExists(envPath))) {
+    throw new Error(`${envPath} does not exist.`);
+  }
+
+  const settings = await readSettings(projectPath);
+  if (!settings.trackingId) {
+    throw new Error("trackingId is missing in settings.json");
```
Comment (swatDong) [inline packages/run-utils/src/index.ts:49]: Consider later:
How to handle decrypt error, throw or ignore?
Or, add one more crypt option to let user to choose whether to decrypt or not...

**Review by swatDong — APPROVED** [review]

File: packages/run-utils/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@microsoft/teamsfx-run-utils",
+  "version": "1.0.0",
+  "description": "Provides the utilities to run TeamsFx project locally.",
+  "main": "./lib/index.js",
+  "scripts": {
+    "clean": "rimraf lib",
+    "build": "rimraf lib && npx tsc -p ./",
```
Comment (swatDong) [inline packages/run-utils/package.json:8]: Consider later - use webpack to bundle all src and node_modules into one single file.

File: packages/run-utils/src/index.ts (hunk 1 above)
Comment (kuojianlu) [reply packages/run-utils/src/index.ts:49]: Sure.

File: packages/run-utils/package.json (hunk 2 above)
Comment (kuojianlu) [reply packages/run-utils/package.json:8]: Sure.

**Pull request merged**